src/js/style.js

```diff
@@ -0,0 +1,27 @@
+"use strict";
+
+import * as Layers from "../layer/index.js";
+
+//Generate style.json
+export function build(tileURL, spriteURL, locales) {
+  return {
+    id: "streets",
```

Comment:
I don’t think an ID is even necessary. This ID might’ve come from a GL JS runtime styling example that used Mapbox Streets or something along those lines.

Reply:
Fixed in 576064f.  Glad we were able to save a few bytes off of our 1MB style file :-D